src/BaseFormModel.php

```
@@ -237,12 +243,31 @@ private function error(): FieldError
 
+    /**
+     * Returns the first matching instantiated property attribute for a field.
+     *
+     * @param string $field Field name to check for the attribute.
+     * @param string $attribute Attribute class name to look for.
+     *
+     * @return object|null Instantiated attribute object if found, or `null` if not found.
+     */
+    private function getFieldPropertyAttribute(string $field, string $attribute): object|null
+    {
+        if (!Reflector::hasProperty($this, $field)) {
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\PHPForge\Helper\Reflector' in method 'getFieldPropertyAttribute'. (undefined)

(StaticAccess)